tests/pds/dataservice_test.go

```diff
@@ -905,12 +905,14 @@ var _ = Describe("{RestartPXPods}", func() {
 
 				defer func() {
 					Step("Delete the workload generating deployments", func() {
-						if ds.Name == "Cassandra" || ds.Name == "PostgreSQL" {
-							err = pdslib.DeleteK8sDeployments(dep.Name, namespace)
-						} else {
-							err = pdslib.DeleteK8sPods(pod.Name, namespace)
+						if !(ds.Name == mysql || ds.Name == kafka || ds.Name == zookeeper) {
```

Comment:
Fix looks good to me, can we add it all places ?

Reply:
done